packages/cli-repl/src/async-repl.spec.ts

```diff
@@ -68,10 +68,9 @@ describe('AsyncRepl', () => {
   it('allows sync interruption through SIGINT', async function() {
     if (process.platform === 'win32') {
       this.skip(); // No SIGINT on Windows.
-      return;
```

Comment:
I just removed the return because the `this.skip()` has return type `never` and it was always highlighted in VSCode as an unreachable line. It still skips everything as it should.

Reply:
(I've made it a consistent `return this.skip()` in those cases 👍  easier to understand when reading and doesn't give a "visible warning")